templates/agent.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "nlweb-dynamic-infrastructure-user.fullname" . }}-agent
+  labels:
+    {{- include "nlweb-dynamic-infrastructure-user.labels" . | nindent 4 }}
```

Comment:
Instead of range I think just this should work:
{{- toYaml .Values.neoload.labels.agent | nindent 4 }}